### PR TITLE
chore: prepare v0.2.5 release

### PR DESCRIPTION
**Pull request by carllerche**
Also includes:
- `tokio-macros` v0.2.1

### CODE REVIEW

File: tokio-macros/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 0.2.1 (December 18, 2019)
+
+### Fixes
+- maintain fn visibility when transforming (#1954).
```
Comment (DoumanAsh) [inline tokio-macros/CHANGELOG.md]: Maybe better wordin would be `Inherit fn visibility when wrapping function`

File: tokio/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+
+### Fixes
+- deadlock risk while shutting down the runtime (#1972).
+- panic while shutting down the runtime (#1978).
+- `sync::MutexGuard` debug output (#1961).
+- misc doc improvements (#1933, #1934, #1940, #1942).
+
+### Changes
+- runtime threads are configured with `runtime::Builder::core_threads` and
+  `runtime::Builder::num_threads`. `runtime::Builder::num_threads` is
```
Comment (DoumanAsh) [inline tokio/CHANGELOG.md]: first `num_threads` -> `max_threads`

Comment (carllerche) [reply tokio/CHANGELOG.md]: woops

File: tokio-macros/CHANGELOG.md (hunk 1 above)
Comment (carllerche) [reply tokio-macros/CHANGELOG.md]: tweaked.

**Comment by carllerche** [comment]
Fixed.

**Pull request merged**